integration_tests/src/main/python/parquet_test.py

```diff
@@ -125,6 +125,15 @@ def test_ts_read_round_trip(spark_tmp_path, ts_write, ts_rebase, small_file_opt,
             conf={'spark.rapids.sql.format.parquet.smallFiles.enabled': small_file_opt,
                   'spark.sql.sources.useV1SourceList': v1_enabled_list})
 
+def test_parquet_write_ts_millis(spark_tmp_path):
+    gen = TimestampGen()
+    data_path = spark_tmp_path + '/PARQUET_DATA'
+    with_gpu_session(
+        lambda spark : unary_op_df(spark, gen).write.parquet(data_path),
+        conf={'spark.sql.parquet.outputTimestampType': 'TIMESTAMP_MILLIS'})
```

Comment:
nit could we parameterize this so in the future when we hopefully support INT96 it is a very small change to test that too?